### DISCUSSION TITLE
Services label translation

### DISCUSSION THREAD

**Discussion by tthom-gh** [orestbida/cookieconsent#651] [Q&A]
Hi! 

It's a very awesome tool, thank you. 
One thing I can't find in the documentation (of course I may have overlooked it) is how to translate the labels of custom services created within categories. Is it even possible? 

I am thinking of this label:

`analytics: {
	services: {
                analytics_storage: {
                    label: 'Google Analytics Storage',`

Thank you!

**Comment by orestbida** [comment]
Unfortunately labels cannot be translated (yet).

**Reply by tthom-gh** [reply]
Thanks for the reply, I will not look for how to do it. :)
I will keep an eye on the change log.

**Comment by webchroma** [comment]
Hi, thanks for the great tool. 

Are there some news on this issue or is there a way I could hook the tool to acchieve this?

thanks piero
